txnkv/txnlock/lock_resolver.go

```diff
@@ -988,7 +1048,7 @@ func (lr *LockResolver) resolveRegionLocks(bo *retry.Backoffer, l *Lock, region
 	return nil
 }
 
-func (lr *LockResolver) resolveLock(bo *retry.Backoffer, l *Lock, status TxnStatus, lite bool, cleanRegions map[locate.RegionVerID]struct{}) error {
+func (lr *LockResolver) resolveLock(bo *retry.Backoffer, l *Lock, status TxnStatus, lite bool, cleanRegions map[locate.RegionVerID]struct{}, async bool) error {
```

Comment:
Where is `async` used?

Reply:
It's used for recording resolve details with different types. Removed because the further details are not included in this PR.